src/System.Diagnostics.Debug/src/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Unix;
       netstandard-Windows_NT;
+      netcoreapp1.2corert-Unix;
+      netcoreapp1.2corert-Windows_NT;
```

Comment:
Yes, but nothing OS specific.  ie, the buildconfigurations entry could just be `netcoreapp1.2corert`, unless there is something intrinsic to GenFacadesArgs which requires an OS be targeted.  From a cursory glance, I don't see anything.